stubs/portpicker/portpicker.pyi

```diff
@@ -0,0 +1,18 @@
+from typing_extensions import TypeAlias
+
+_Port: TypeAlias = int
+
+class NoFreePortFoundError(Exception): ...
+
+def add_reserved_port(port: _Port) -> None: ...
+def return_port(port: _Port) -> None: ...
+def bind(port: _Port, socket_type: int, socket_proto: int) -> _Port | None: ...
```

Comment:
We could possibly be slightly stricter here, and say that `socket_type` has to be an instance of `socket.SocketKind`? The runtime accepts an `int`, but `socket.SocketKind` is more precise -- any member of that enum is guaranteed to work for this function, whereas it's not the case that any `int` is guaranteed to work

Reply:
Good point, changing.